mantidproject/mantid
# ISISDirectInelastic failing system tests for Python 3

The way the `Ei` value is formatted in the SampleLogs has been changed from using the `str()` function (which has a differing precision between python2.7 and python3.x) to the `format()` function which  explicitly quotes the precision.

The reference file were also found to not contain the correct precision for `Ei` so they have also been updated (See #20440 for more details on ref file update). @abuts would you be able to take a look at the change to the reference files and check that you are happy with this?

**To test:**

Ensure the system tests for:
* `ISISDirectInelastic.MARIReductionAutoEi`
* `ISISDirectInelastic.MARIReductionMonSeparate`
* `ISISDirectInelastic.MAPSDgreduceReduction`

pass on all platforms (**including Python 3**)

Note: If you are testing this locally, the reference files have been updated so you may need to run CMake again.

Fixes #20440 

**Release Notes** 
Internal change to system tests hence not required in the release notes.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
